conf/postgres-operator/exporter.json

```diff
@@ -0,0 +1,58 @@
+,{
```

Comment:
I think this file needs to be moved to the consolidated installation files?

Reply:
Yes, similar to the issue above. Updated.